command/connect/envoy/envoy.go

```diff
@@ -258,13 +263,19 @@ func (c *cmd) templateArgs() (*BootstrapTplArgs, error) {
 		cluster = c.sidecarFor
 	}
 
+	adminAccessLogPath := c.adminAccessLogPath
+	if len(adminAccessLogPath) == 0 {
```

Comment:
Would this be a little clearer with `if adminAccessLogPath == ""`? Since it's a string.

Reply:
I always wonder about the same thing and I read the first answer of https://stackoverflow.com/questions/18594330/what-is-the-best-way-to-test-for-an-empty-string-in-go. If you find `== ""` better, I would be happy to change it. I don't have an opinion on this one.

Reply:
I think `== ""` would be clearer and more consistent. I did a search through the Consul repo and found that `len(x) == 0)` didn't seem to ever be used with strings. 

Thanks!